.gitignore

```diff
@@ -1,10 +1,3 @@
-# Ignore all hidden files
-.*
-
-# Include travis.yml
-\!.travis.yml
-\!.flake8
```

Comment:
I think it's okay to allow hidden files. If we see this as an issue we can revisit. It was a larger issue before when we were planning on doc files living within the repo that were auto generated.